2200063/민석이의_과제_체크하기.py

```diff
@@ -0,0 +1,14 @@
+T = int(input())
+
+for t in range(T):
+    N, K = map(int, input().split())
+    P = list(map(int, input().split()))
+    result = []
+
+    for i in range(1, N+1):
+        if i not in P:
+            result.append(i)
+    
+    result = sorted(result)
```

Comment:
완벽합니다! 💯 
```suggestion
    for i in range(1, N+1):
        if i not in P:
            result.append(i)

```
이미 `range`에서 정렬된 순서로 탐색을 하므로, `sort` 해주지 않으셔도 괜찮습니다 😄